docs/glsl-fuzz-walkthrough.md

```diff
@@ -858,11 +981,37 @@ Explanation:
   * If the worker fails to respond twice, assume the shader fails to render.
 * `--seed -136936935`
   * Random seed for the reduction algorithm.
-* `--server http://localhost:8080 --worker-name my-worker`
+* `--server http://localhost:8080 --worker my-worker`
   * `glsl-reduce` will, by default, run shader jobs locally, which is not well-tested.
 We specify a server and worker name to run the shader jobs on a worker that is connected
 to the server.
 
+
+### Performing a bad result reduction for a compute shader
+
+If you have a variant compute shader that produces a different SSBO result compared with its associated reference, you can reduce the problem as follows.
+
+```sh
+# Copy the .info.json file for the reference into your working directory
+cp /path/to/reference.info.json .
+# Make a copy of the template interestingness test for compute shader differences
+cp graphicsfuzz/python/interestingness/compute_interesting_diff.py .
```

Comment:
The chmod should not be necessary. I believe the file is made executable before being added to the zip. The instructions here are unlikely to work on Windows even if they use a bash shell because the reducer cannot launch `./compute_interesting_diff.py` on Windows; we should include bash and cmd wrappers and copy them too. 

So I suggest the following:

* Remove the chmod command from the docs.
* Make the python scripts executable before committing; it is not necessary, but it is good to be consistent. 
* Add bash and cmd wrappers for the scripts in `interestingness`. 
* Docs: change the `cp` to include the appropriate wrappers. 
* Docs: change the glsl-reduce command to remove `.py`; use `./compute_interesting_diff`. 
* Same for `compute_interesting_same.py`.

Reply:
Done.